pytensor/tensor/nlinalg.py

```diff
@@ -1026,3 +1027,45 @@ def tensorsolve(a, b, axes=None):
     "tensorinv",
     "tensorsolve",
 ]
+
+
+# Adding the kron function here, which earlier used to be in slinalg.py
```

Comment:
No need for this comment, if someone is super interested he can check the git history